config/initializers/okcomputer.rb

```diff
@@ -15,11 +64,11 @@
 OkComputer::Registry.register 'solr_url', OkComputer::HttpCheck.new(solr_url)
 
 # fedora
-OkComputer::Registry.register 'fedora_url', OkComputer::HttpCheck.new(Dor::Config.fedora.url)
+OkComputer::Registry.register 'fedora_url', HttpWithClientKeyCheck.new(Dor::Config.fedora.url)
 
 # suri
 suri_url = Dor::Config.suri.url + '/suri2/namespaces'
-OkComputer::Registry.register 'suri_url', OkComputer::HttpCheck.new(suri_url)
+OkComputer::Registry.register 'suri_url', HttpWithClientKeyCheck.new(suri_url)
```

Comment:
You are correct. I wasn't properly failing on an error code of != 200, so this appeared to work.  I fixed it now.